### ISSUE TITLE
Deeply nested infobox

### ISSUE THREAD

**Issue by yanosh-igor** [spencermountain/wtf_wikipedia#264]
Topic Spain
https://en.wikipedia.org/wiki/Spain
The text contains wiki markup.

![2019-04-04_1450](https://user-images.githubusercontent.com/435827/55553562-2df65d80-56e9-11e9-928d-47138f732d38.png)

**Comment by spencermountain** [comment]
yikes
took a look briefly - 
```
country infobox
   → collapsible list
     → another unnamed infobox
```
likely a dupe of #247 
hopefully that will fix it -
cheers

**Comment by spencermountain** [comment]
merging this with #247

**Comment by yanosh-igor** [comment]
@spencermountain Hey, unfortunately, it wasn't fixed 
Another example is https://en.wikipedia.org/wiki/Ukraine

![2019-10-03_1234](https://user-images.githubusercontent.com/435827/66115793-3406c180-e5da-11e9-8fd6-d40eb9d2d562.png)

**Comment by spencermountain** [comment]
yeah, i've just merged it with #247 it is still open.
